public/docs/dart/latest/guide/dependency-injection.jade

```diff
@@ -1,6 +1,791 @@
 include ../../../../_includes/_util-fns
+:marked
```

Comment:
We need to revert the changes to this file (though I'll definitely use your diffs to guide my changes).